twitterscraper/query.py

```diff
@@ -159,10 +160,6 @@ def query_tweets_once(*args, **kwargs):
 
 def query_tweets(query, limit=None, begindate=dt.date(2006,3,21), enddate=dt.date.today(), poolsize=20, lang=''):
     no_days = (enddate - begindate).days
-
```

Comment:
This was implemented to fix an error which occurs when the begindate  is later than the enddate. See https://github.com/taspinar/twitterscraper/pull/143. 
Is there a reason why this piece of code is removed?